### PR TITLE
Fix bug, refactor

### PR DESCRIPTION
**Pull request by AndrewHelix**

### CODE REVIEW

File: src/components/feedback/ViewFeedbacks.tsx [hunk 1]
```diff
@@ -13,22 +13,28 @@ export const ViewFeedbacks: FC<ViewFeedbackProps> = ({ feedbacks }) => {
...
-                key={value}
-                className={styles['rating-label_feedbacks']}
-              />
-            ))}
-          </div>
-          <p className={styles['feedback-article-text']}>{feedback}</p>
-        </article>
-      ))}
+      {feedbacks.length ? (
+        <>
```
Comment (JuliaShavyrova) [inline src/components/feedback/ViewFeedbacks.tsx]: redundant fragment tag)

File: src/components/header/Header.tsx [hunk 2]
```diff
@@ -62,6 +62,13 @@ const Header: FC = () => {
     setIsShowMenu(!isShowMenu);
   };
 
+  const setFalse = () => setIsShowMenu(false);
```
Comment (JuliaShavyrova) [inline src/components/header/Header.tsx]: please rename, `setFalse` doesn't describe what exactly function does

File: src/context/positionContext.tsx [hunk 3]
```diff
@@ -67,7 +67,7 @@ export const PositionContextProvider = ({ children }: { children: ReactNode }) =
   return (
     <Provider value={{ state, changePosition }}>
       <>
-        {(latitude || longitude) && children}
+        {latitude || longitude ? children : null}
```
Comment (JuliaShavyrova) [inline src/context/positionContext.tsx:70]: don't you need brackets here?

File: src/components/feedback/ViewFeedbacks.tsx (hunk 1 above)
Comment (AndrewHelix) [reply src/components/feedback/ViewFeedbacks.tsx]: without fragment I have error
image

File: src/context/positionContext.tsx (hunk 3 above)
Comment (AndrewHelix) [reply src/context/positionContext.tsx:70]: it work correct without brackets

File: src/components/feedback/ViewFeedbacks.tsx (hunk 1 above)
Comment (JuliaShavyrova) [reply src/components/feedback/ViewFeedbacks.tsx]: you need to remove extra brackets:
`  {feedbacks.length ? feedbacks.map(({ id, name, rating, feedback }) => (`

File: src/components/header/Header.tsx [hunk 4]
```diff
@@ -62,6 +62,13 @@ const Header: FC = () => {
     setIsShowMenu(!isShowMenu);
   };
 
+  const setFalseInIsShowState = () => setIsShowMenu(false);
```
Comment (JuliaShavyrova) [inline src/components/header/Header.tsx]: FYI: `hideMenu/closeMenu` would be enough :)

**Review by JuliaShavyrova — APPROVED** [review]

**Pull request merged**